ci/go.sum

```diff
@@ -1,7 +1,5 @@
-github.com/gocarina/gocsv v0.0.0-20231116093920-b87c2d0e983a h1:RYfmiM0zluBJOiPDJseKLEN4BapJ42uSi9SZBQ2YyiA=
-github.com/gocarina/gocsv v0.0.0-20231116093920-b87c2d0e983a/go.mod h1:5YoVOkjYAQumqlV356Hj3xeYh4BdZuLE0/nRkf2NKkI=
-dagger.io/dagger v0.10.0 h1:iRzQLVOe89rZQwfpcKq8kZLBLXU50uiDti7txZjcl6A=
-dagger.io/dagger v0.10.0/go.mod h1:aq9P87v4apEOXKPXzn35uK+by5dUplSU5ZeXm+E8Sgw=
+dagger.io/dagger v0.10.2 h1:8q4AwKm48qAWZdY7O2NMZ3FveteDHJeM3WwmcNKZOM0=
```

Comment:
Out of pure curiosity, how come /gocarina is no longer needed?

Reply:
Good question. Idk xD I'm guessing it may have been a package we used at one point and then deleted from imports thus it being gone. Either way should be fine. The go.sum file is only used by your own compiler and is even generated as you run go mod tidy and such so shouldn't matter too much :0

Reply:
Ah, alrighty c: thank you, and good to know!